### PR TITLE
Adding simple Traefik example.

### PR DESCRIPTION
**Pull request by lfache**
Just add a simple Traefik reverse proxy example like nginx golang

### CODE REVIEW

File: traefik-golang/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+  frontend:
+    image: traefik:2.2
+    command: --providers.docker --entrypoints.web.address=:80 --providers.docker.exposedbydefault=false
```
Comment (glours) [inline traefik-golang/docker-compose.yml:5]: ENV variables seems to be a better choice, they can be overridden, WDYT?

Comment (ldez) [reply traefik-golang/docker-compose.yml:5]: env vars is the worse choice, they cannot be overridden after the start of Traefik.

The best choice is the CLI flags.

Comment (lfache) [reply traefik-golang/docker-compose.yml:5]: Done !
And I think using a traefik.yml or toml wouldn't be that easy. I wanted to use the simplest possible solution.

**Comment by glours** [comment]
@lfache I just need you fix the DCO issue to merge your PR
Thank you very much for your contribution 👍

**Comment by glours** [comment]
Arff now there's a conflit in the README.md, can you rebase with master please?

**Comment by glours** [comment]
Thanks 🙏

**Pull request merged**